src/cider/function/internals/functions_string.yaml

```diff
@@ -1511,6 +1511,59 @@ scalar_functions:
           - value: i64
             name: "group"
         return: "string"
+  - 
+    name: "in"
```

Comment:
Q: Does this mean we need to maintain sth similar to the upstream yaml moving forward? https://github.com/substrait-io/substrait/commit/81e34d4054ff0dbde23ac749fbb8fcc232989c5d#diff-b05c613f1c6a478bd62cd87456ade235540725c12afced21e5c3d534cd1da1e5R748

Reply:
Yes, we need to update the yaml file when cider add new feature.